make install ignores DESTDIR variable

**Describe the bug**
In Arch Linux I am building Arkime from source. The building occurs in a clean fakeroot directory. It builds correctly and I need to do install it in a specific directory. The building occurs by:
```
  ./bootstrap.sh
  ./configure \
    --prefix=/usr/share/arkime \
    --bindir=/usr/bin \
    --with-zstd=yes \
    --with-libpcap=no \
    --with-yara=no \
    --with-glib2=no \
    --with-pfring=no \
    --with-curl=no \
    --with-lua=no LIBS="-lpcap -lyara -llua -lcurl" GLIB2_CFLAGS="-I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include" GLIB2_LIBS="-lglib-2.0 -lgmodule-2.0 -lgobject-2.0 -lgio-2.0" \
    KAFKA_LIBS="-lrdkafka" KAFKA_CFLAGS="-I/usr/include/librdkafka" \
    --with-kafka=no

make
```
And I install it by:
```
make DESTDIR="$pkgdir" install
```
where `$pkgdir` is the clean fakeroot directory corresponding to `/tmp/makepkg/arkime/pkg/arkime`. So, I expect that, the usage of the prefix `/usr/share/arkime` should then install files in `/tmp/makepkg/arkime/pkg/arkime/usr/share/arkime`, but it does not happen, indeed, `make DESTDIR="$pkgdir" install` returns the following error:
```

added 1954 packages, and audited 1955 packages in 33s

197 packages are looking for funding
  run `npm fund` for details

23 vulnerabilities (9 low, 12 moderate, 2 high)

To address issues that do not require attention, run:
  npm audit fix

To address all issues possible (including breaking changes), run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
cp -pr common /usr/share/arkime
cp: cannot create directory '/usr/share/arkime': Permission denied
make[2]: *** [Makefile:879: install-exec-local] Error 1
make[2]: Leaving directory '/tmp/makepkg/arkime/src/arkime-5.6.0'
make[1]: *** [Makefile:763: install-am] Error 2
make[1]: Leaving directory '/tmp/makepkg/arkime/src/arkime-5.6.0'
make: *** [Makefile:456: install-recursive] Error 1
==> ERROR: A failure occurred in package().
    Aborting...
```
So, the error `cp: cannot create directory '/usr/share/arkime': Permission denied` is legit because since we are building in a fakeroot directory, we cannot have permission outside it. The correct approach is that the `make install` recognizes `DESTDIR=$pkgdir` variable and copies files and dirs like `common` to `/tmp/makepkg/arkime/pkg/arkime/usr/share/arkime`.

**Elasticsearch version**: 7.10.2

**Arkime/Moloch version**: 5.6.0

**OS name and version**: Arch Linux rolling

**Node version**: 20.18.1

--- a/Makefile.am
+++ b/Makefile.am
@@ -19,16 +19,17 @@ endif
 SUBDIRS += tests
 
 install-exec-local:
+	$(MKDIR_P) $(DESTDIR)@prefix@
 	npm ci
-	cp -pr common @prefix@
-	cp -pr assets @prefix@
-	@INSTALL@ package.json @prefix@/package.json
-	@INSTALL@ package-lock.json @prefix@/package-lock.json
-	(cd @prefix@ ; npm ci --production)
-	rm -f @prefix@/package-lock.json
+	cp -pr common $(DESTDIR)@prefix@
+	cp -pr assets $(DESTDIR)@prefix@ 
+	@INSTALL@ package.json $(DESTDIR)@prefix@/package.json
+	@INSTALL@ package-lock.json $(DESTDIR)@prefix@/package-lock.json
+	(cd $(DESTDIR)@prefix@ ; npm ci --production)
+	rm -f $(DESTDIR)@prefix@/package-lock.json
 
 check-local:
 	npm ci
 
 config:
-	@prefix@/bin/Configure
+	$(DESTDIR)@prefix@/bin/Configure

--- a/capture/Makefile.in
+++ b/capture/Makefile.in
@@ -41,8 +41,8 @@ C_FILES         = main.c db.c yara.c http.c config.c parsers.c plugins.c field.c
 O_FILES         = $(C_FILES:.c=.o)
 
 INSTALL         = @INSTALL@
-bindir          = @prefix@/bin
-installinclude  = @prefix@/include
+bindir          = $(DESTDIR)@prefix@/bin
+installinclude  = $(DESTDIR)@prefix@/include
 
 SANITIZE_LDFLAGS = -fno-common -fsanitize=address -fsanitize=integer -fsanitize=nullability
 SANITIZE_CFLAGS  = $(SANITIZE_LDFLAGS)
@@ -92,12 +92,12 @@ thirdparty/http_parser.o: thirdparty/http_parser.c
 	$(CC) -fno-strict-aliasing -pthread -fPIC -DNDEBUG -DHTTP_PARSER_STRICT=0 -DHTTP_PARSER_DEBUG=0 @CFLAGS@ -c thirdparty/http_parser.c -o thirdparty/http_parser.o
 
 install: installdirs
-	$(INSTALL) *.h thirdparty/http_parser.h $(DESTDIR)$(installinclude)
-	$(INSTALL) capture $(DESTDIR)$(bindir)/capture
+	$(INSTALL) *.h thirdparty/http_parser.h $(installinclude)
+	$(INSTALL) capture $(bindir)/capture
 
 installdirs:
-	$(MKDIR_P) $(DESTDIR)$(installinclude)
-	$(INSTALL) -d $(DESTDIR)$(bindir)
+	$(MKDIR_P) $(installinclude)
+	$(INSTALL) -d $(bindir)
 	(cd parsers; $(MAKE) install)
 	(cd plugins; $(MAKE) install)
 

--- a/capture/Makefile.in
+++ b/capture/Makefile.in
@@ -1,3 +1,4 @@
+MKDIR_P = mkdir -p
 INSTALL_DIR   = @INSTALL_DIR@
 
 # If ARKIME_BUILD_FULL_VERSION is set, use it
@@ -91,12 +92,12 @@ thirdparty/http_parser.o: thirdparty/http_parser.c
 	$(CC) -fno-strict-aliasing -pthread -fPIC -DNDEBUG -DHTTP_PARSER_STRICT=0 -DHTTP_PARSER_DEBUG=0 @CFLAGS@ -c thirdparty/http_parser.c -o thirdparty/http_parser.o
 
 install: installdirs
-	@mkdir -p $(installinclude)
-	$(INSTALL) *.h thirdparty/http_parser.h $(installinclude)
-	$(INSTALL) capture $(bindir)/capture
+	$(INSTALL) *.h thirdparty/http_parser.h $(DESTDIR)$(installinclude)
+	$(INSTALL) capture $(DESTDIR)$(bindir)/capture
 
 installdirs:
-	$(INSTALL) -d $(bindir)
+	$(MKDIR_P) $(DESTDIR)$(installinclude)
+	$(INSTALL) -d $(DESTDIR)$(bindir)
 	(cd parsers; $(MAKE) install)
 	(cd plugins; $(MAKE) install)
 

--- a/capture/parsers/Makefile.in
+++ b/capture/parsers/Makefile.in
@@ -6,8 +6,7 @@ INCLUDE_OTHER = -I.. -I../thirdparty \
 	        @NGHTTP2_CFLAGS@ \
                 @GLIB2_CFLAGS@
 
-install_sh = @install_sh@
-mkdir_p = @mkdir_p@
+MKDIR_P = mkdir -p
 INSTALL = @INSTALL@
 PARSERSDIR = $(DESTDIR)@prefix@/parsers
 
@@ -20,7 +19,7 @@ SOS=$(SRCS:.c=.so)
 all:$(SOS)
 
 install:
-	@mkdir -p "$(PARSERSDIR)"
+	$(MKDIR_P) "$(PARSERSDIR)"
 	$(INSTALL) *.so *.jade $(PARSERSDIR)
 
 distclean realclean clean:

--- a/capture/parsers/Makefile.in
+++ b/capture/parsers/Makefile.in
@@ -9,7 +9,7 @@ INCLUDE_OTHER = -I.. -I../thirdparty \
 install_sh = @install_sh@
 mkdir_p = @mkdir_p@
 INSTALL = @INSTALL@
-PARSERSDIR = @prefix@/parsers
+PARSERSDIR = $(DESTDIR)@prefix@/parsers
 
 SRCS=$(wildcard *.c)
 SOS=$(SRCS:.c=.so)

--- a/capture/plugins/Makefile.in
+++ b/capture/plugins/Makefile.in
@@ -9,8 +9,8 @@ INCLUDE_OTHER = -I.. -I../thirdparty \
 install_sh = @install_sh@
 mkdir_p = @mkdir_p@
 INSTALL = @INSTALL@
-PLUGINDIR = @prefix@/plugins
-BINDIR = @prefix@/bin
+PLUGINDIR = $(DESTDIR)@prefix@/plugins
+BINDIR = $(DESTDIR)@prefix@/bin
 
 SRCS=$(wildcard *.c)
 SOS=$(SRCS:.c=.so)

--- a/capture/plugins/Makefile.in
+++ b/capture/plugins/Makefile.in
@@ -6,8 +6,7 @@ INCLUDE_OTHER = -I.. -I../thirdparty \
 	        @ZSTD_CFLAGS@ \
                 @GLIB2_CFLAGS@
 
-install_sh = @install_sh@
-mkdir_p = @mkdir_p@
+MKDIR_P = mkdir -p
 INSTALL = @INSTALL@
 PLUGINDIR = $(DESTDIR)@prefix@/plugins
 BINDIR = $(DESTDIR)@prefix@/bin
@@ -24,9 +23,9 @@ check:
 
 install:
 	$(INSTALL) taggerUpload.pl $(BINDIR)
-	@mkdir -p "$(PLUGINDIR)"
+	$(MKDIR_P) "$(PLUGINDIR)"
 	$(INSTALL) *.so *.jade *.js $(PLUGINDIR)
-	@mkdir -p "$(PLUGINDIR)/writer-s3"
+	$(MKDIR_P) "$(PLUGINDIR)/writer-s3"
 	$(INSTALL) writer-s3/*.js $(PLUGINDIR)/writer-s3
 
 distclean realclean clean:

--- a/capture/plugins/daq/Makefile.in
+++ b/capture/plugins/daq/Makefile.in
@@ -3,12 +3,6 @@ INCLUDE_PCAP  = @PCAP_CFLAGS@
 INCLUDE_OTHER = -I../.. -I../../thirdparty \
                 @GLIB2_CFLAGS@
 
-install_sh = @install_sh@
-mkdir_p = @mkdir_p@
-INSTALL = @INSTALL@
-PLUGINDIR = @prefix@/plugins
-WISEDIR = @prefix@/wiseService
-
 SRCS=$(wildcard *.c)
 SOS=$(patsubst %.c,../%.so,$(SRCS))
 

--- a/capture/plugins/kafka/Makefile.in
+++ b/capture/plugins/kafka/Makefile.in
@@ -3,11 +3,6 @@ LIB_KAFKA = @KAFKA_LIBS@
 INCLUDE_OTHER = -I../.. -I../../thirdparty \
                 @GLIB2_CFLAGS@
 
-install_sh = @install_sh@
-mkdir_p = @mkdir_p@
-INSTALL = @INSTALL@
-PLUGINDIR = @prefix@/plugins
-
 SRCS=$(wildcard *.c)
 SOS=$(patsubst %.c,../%.so,$(SRCS))
 

--- a/capture/plugins/lua/Makefile.in
+++ b/capture/plugins/lua/Makefile.in
@@ -1,10 +1,6 @@
 INCLUDE_OTHER = -I../.. -I../../thirdparty \
                 @GLIB2_CFLAGS@
 
-install_sh = @install_sh@
-mkdir_p = @mkdir_p@
-INSTALL = @INSTALL@
-PLUGINDIR = @prefix@/plugins
 LUA_CFLAGS = @LUA_CFLAGS@
 LUA_LIBS = @LUA_LIBS@
 

--- a/capture/plugins/pfring/Makefile.in
+++ b/capture/plugins/pfring/Makefile.in
@@ -3,12 +3,6 @@ INCLUDE_PCAP  = @PCAP_CFLAGS@
 INCLUDE_OTHER = -I../.. -I../../thirdparty \
                 @GLIB2_CFLAGS@
 
-install_sh = @install_sh@
-mkdir_p = @mkdir_p@
-INSTALL = @INSTALL@
-PLUGINDIR = @prefix@/plugins
-WISEDIR = @prefix@/wiseService
-
 SRCS=$(wildcard *.c)
 SOS=$(patsubst %.c,../%.so,$(SRCS))
 

--- a/capture/plugins/snf/Makefile.in
+++ b/capture/plugins/snf/Makefile.in
@@ -3,11 +3,6 @@ INCLUDE_PCAP  = @PCAP_CFLAGS@
 INCLUDE_OTHER = -I../.. -I../../thirdparty \
                 @GLIB2_CFLAGS@
 
-install_sh = @install_sh@
-mkdir_p = @mkdir_p@
-INSTALL = @INSTALL@
-PLUGINDIR = @prefix@/plugins
-WISEDIR = @prefix@/wiseService
 SNFDIR = /opt/snf
 
 SRCS=$(wildcard *.c)

--- a/cont3xt/Makefile.in
+++ b/cont3xt/Makefile.in
@@ -1,8 +1,7 @@
 install_sh = @install_sh@
-mkdir_p = @mkdir_p@
+MKDIR_P = mkdir -p
 INSTALL = @INSTALL@
-CONT3XTDIR = @prefix@/cont3xt
-BINDIR = @prefix@/bin
+CONT3XTDIR = $(DESTDIR)@prefix@/cont3xt
 CP = /bin/cp
 INSTALL_BUNDLE ?= "bundle:min"
 
@@ -14,7 +13,7 @@ check:
 	npm run bundle
 
 install:
-	@mkdir -p "$(CONT3XTDIR)" "$(CONT3XTDIR)/vueapp" "$(CONT3XTDIR)/integrations"
+	$(MKDIR_P) "$(CONT3XTDIR)" "$(CONT3XTDIR)/vueapp" "$(CONT3XTDIR)/integrations"
 	$(INSTALL) *.js package.json package-lock.json $(CONT3XTDIR)
 	npm ci
 	npm run $(INSTALL_BUNDLE)

--- a/cont3xt/Makefile.in
+++ b/cont3xt/Makefile.in
@@ -1,4 +1,3 @@
-install_sh = @install_sh@
 MKDIR_P = mkdir -p
 INSTALL = @INSTALL@
 CONT3XTDIR = $(DESTDIR)@prefix@/cont3xt

--- a/db/Makefile.in
+++ b/db/Makefile.in
@@ -1,5 +1,4 @@
-install_sh = @install_sh@
-mkdir_p = @mkdir_p@
+MKDIR_P = mkdir -p
 INSTALL = @INSTALL@
 DBDIR = $(DESTDIR)@prefix@/db
 
@@ -7,7 +6,7 @@ DBDIR = $(DESTDIR)@prefix@/db
 all:
 
 install:
-	@mkdir -p "$(DBDIR)"
+	$(MKDIR_P) "$(DBDIR)"
 	$(INSTALL) db.pl $(DBDIR)
 
 check:

--- a/db/Makefile.in
+++ b/db/Makefile.in
@@ -1,7 +1,7 @@
 install_sh = @install_sh@
 mkdir_p = @mkdir_p@
 INSTALL = @INSTALL@
-DBDIR = @prefix@/db
+DBDIR = $(DESTDIR)@prefix@/db
 
 
 all:

--- a/parliament/Makefile.in
+++ b/parliament/Makefile.in
@@ -1,4 +1,3 @@
-install_sh = @install_sh@
 MKDIR_P = mkdir -p
 INSTALL = @INSTALL@
 PARLIAMENTDIR = $(DESTDIR)@prefix@/parliament

--- a/parliament/Makefile.in
+++ b/parliament/Makefile.in
@@ -1,7 +1,7 @@
 install_sh = @install_sh@
-mkdir_p = @mkdir_p@
+MKDIR_P = mkdir -p
 INSTALL = @INSTALL@
-PARLIAMENTDIR = @prefix@/parliament
+PARLIAMENTDIR = $(DESTDIR)@prefix@/parliament
 CP = /bin/cp
 
 
@@ -12,7 +12,7 @@ check:
 	npm run bundle
 
 install:
-	@mkdir -p "$(PARLIAMENTDIR)" "$(PARLIAMENTDIR)/vueapp"
+	$(MKDIR_P) "$(PARLIAMENTDIR)" "$(PARLIAMENTDIR)/vueapp"
 	$(INSTALL) *.js package.json package-lock.json $(PARLIAMENTDIR)
 	npm ci
 	npm run bundle:min

--- a/release/Makefile.in
+++ b/release/Makefile.in
@@ -1,14 +1,15 @@
 INSTALL        = @INSTALL@
-etcdir         = @prefix@/etc
+etcdir         = $(DESTDIR)@prefix@/etc
+bindir         = $(DESTDIR)@prefix@/bin
 
 all:
 clean:
 check:
 
 install:
-	$(INSTALL) README.txt @prefix@
-	$(INSTALL) ../LICENSE @prefix@
-	@mkdir -p $(etcdir) @prefix@/logs
+	$(INSTALL) README.txt $(DESTDIR)@prefix@
+	$(INSTALL) ../LICENSE $(DESTDIR)@prefix@
+	@mkdir -p $(etcdir) $(DESTDIR)@prefix@/logs
 	$(INSTALL) config.ini.sample $(etcdir)/
 	$(INSTALL) wise.ini.sample $(etcdir)/
 	$(INSTALL) cont3xt.ini.sample $(etcdir)/
@@ -16,26 +17,26 @@ install:
 	$(INSTALL) parliament.env.example $(etcdir)/
 	$(INSTALL) env.example $(etcdir)/
 
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkimecont3xt.systemd.service > @prefix@/etc/arkimecont3xt.systemd.service
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkimecapture.systemd.service > @prefix@/etc/arkimecapture.systemd.service
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkimeviewer.systemd.service > @prefix@/etc/arkimeviewer.systemd.service
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkimewise.systemd.service > @prefix@/etc/arkimewise.systemd.service
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkimeparliament.systemd.service > @prefix@/etc/arkimeparliament.systemd.service
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkimecont3xt.systemd.service > $(etcdir)/arkimecont3xt.systemd.service
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkimecapture.systemd.service > $(etcdir)/arkimecapture.systemd.service
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkimeviewer.systemd.service > $(etcdir)/arkimeviewer.systemd.service
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkimewise.systemd.service > $(etcdir)/arkimewise.systemd.service
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkimeparliament.systemd.service > $(etcdir)/arkimeparliament.systemd.service
 
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_update_geo.sh > @prefix@/bin/arkime_update_geo.sh
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_add_user.sh > @prefix@/bin/arkime_add_user.sh
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_config_interfaces.sh > @prefix@/bin/arkime_config_interfaces.sh
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < Configure > @prefix@/bin/Configure
-	chmod 0755 @prefix@/bin/arkime_update_geo.sh @prefix@/bin/arkime_add_user.sh @prefix@/bin/Configure @prefix@/bin/arkime_config_interfaces.sh
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_update_geo.sh > $(bindir)/arkime_update_geo.sh
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_add_user.sh > $(bindir)/arkime_add_user.sh
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_config_interfaces.sh > $(bindir)/arkime_config_interfaces.sh
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < Configure > $(bindir)/Configure
+	chmod 0755 $(bindir)/arkime_update_geo.sh $(bindir)/arkime_add_user.sh $(bindir)/Configure $(bindir)/arkime_config_interfaces.sh
 	/bin/bash ./afterinstall.sh
 
 installmoloch:
-	rm -f @prefix@/etc/arkime*
-	rm -f @prefix@/bin/arkime*
+	rm -f $(etcdir)/arkime*
+	rm -f $(bindir)/arkime*
 	$(INSTALL) README.txt @prefix@
 	$(INSTALL) ../LICENSE @prefix@
 	@mkdir -p $(etcdir) @prefix@/logs
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_update_geo.sh > @prefix@/bin/moloch_update_geo.sh
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_add_user.sh > @prefix@/bin/moloch_add_user.sh
-	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < Configure.moloch > @prefix@/bin/Configure
-	chmod 0755 @prefix@/bin/moloch_update_geo.sh @prefix@/bin/moloch_add_user.sh @prefix@/bin/Configure
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_update_geo.sh > $(bindir)/moloch_update_geo.sh
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_add_user.sh > $(bindir)/moloch_add_user.sh
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < Configure.moloch > $(bindir)/Configure
+	chmod 0755 $(bindir)/moloch_update_geo.sh $(bindir)/moloch_add_user.sh $(bindir)/Configure

--- a/viewer/Makefile.in
+++ b/viewer/Makefile.in
@@ -1,4 +1,3 @@
-install_sh = @install_sh@
 INSTALL = @INSTALL@
 MKDIR_P = mkdir -p
 VIEWERDIR = $(DESTDIR)@prefix@/viewer

--- a/viewer/Makefile.in
+++ b/viewer/Makefile.in
@@ -1,7 +1,7 @@
 install_sh = @install_sh@
-mkdir_p = @mkdir_p@
 INSTALL = @INSTALL@
-VIEWERDIR = @prefix@/viewer
+MKDIR_P = mkdir -p
+VIEWERDIR = $(DESTDIR)@prefix@/viewer
 CP = /bin/cp
 INSTALL_BUNDLE ?= "bundle:min"
 CYBERCHEF_VERSION=10.19.2
@@ -20,7 +20,7 @@ cyberchef:
 	(cd public ; if [ ! -f CyberChef_v$(CYBERCHEF_VERSION).zip ]; then wget -nv -N https://github.com/gchq/CyberChef/releases/download/v$(CYBERCHEF_VERSION)/CyberChef_v$(CYBERCHEF_VERSION).zip; fi)
 
 install: cyberchef
-	@mkdir -p "$(VIEWERDIR)" "$(VIEWERDIR)/vueapp"
+	$(MKDIR_P) "$(VIEWERDIR)" "$(VIEWERDIR)/vueapp"
 	/bin/rm -f $(VIEWERDIR)/public/style.css
 	$(INSTALL) *.js package.json package-lock.json $(VIEWERDIR)
 	npm ci

--- a/wiseService/Makefile.in
+++ b/wiseService/Makefile.in
@@ -1,4 +1,3 @@
-install_sh = @install_sh@
 MKDIR_P = mkdir -p
 INSTALL = @INSTALL@
 WISEDIR = $(DESTDIR)@prefix@/wiseService

--- a/wiseService/Makefile.in
+++ b/wiseService/Makefile.in
@@ -1,9 +1,7 @@
 install_sh = @install_sh@
-mkdir_p = @mkdir_p@
+MKDIR_P = mkdir -p
 INSTALL = @INSTALL@
-PLUGINDIR = @prefix@/plugins
-WISEDIR = @prefix@/wiseService
-BINDIR = @prefix@/bin
+WISEDIR = $(DESTDIR)@prefix@/wiseService
 CP = /bin/cp
 
 all:
@@ -14,7 +12,7 @@ check:
 	npm run bundle
 
 install:
-	@mkdir -p "$(WISEDIR)" "$(WISEDIR)/vueapp"
+	$(MKDIR_P) "$(WISEDIR)" "$(WISEDIR)/vueapp"
 	$(INSTALL) *.js wiseService.ini.sample package.json package-lock.json $(WISEDIR)
 	npm ci
 	npm run bundle:min

